app/webpacker/stylesheets/structure/_left-menu.scss

```diff
@@ -1,9 +1,9 @@
-// 
+//
 // menu.scss
 //
 
 body.enlarged .logo{
-  content:url(asset_path("logos/logo_small.svg"));
```

Comment:
I don't understand what we're doing with that line, but it seems to still work once migrated.

here is how it looks right now

<img width="1552" alt="Screenshot 2020-04-16 at 16 25 39" src="https://user-images.githubusercontent.com/883348/79468376-25747a80-7fff-11ea-8472-85e9e6877a2e.png">

Reply:
<img width="800" alt="Capture d’écran 2020-04-16 à 16 36 04" src="https://user-images.githubusercontent.com/29787388/79469400-67ea8700-8000-11ea-8908-253289926fd1.png">
I don't have the same logo on demo ?

Reply:
oh right thanks for checking. you're right it doesn't behave the same on chrome and FF. and there are a lot of different width breakpoints